portfolio/src/main/webapp/script.js

```diff
@@ -60,4 +223,20 @@ var unusedFunFacts =
     'Currently, I am attempting to grow potatoes outside.',
     'My favorite animal is a panda.'];
 
-var countTableEntries = 0;
+var commentTableText = `<table id="comment-table"> 
+            <tr> 
+                <th>Comments</th> 
+            </tr>`;
+
+var commentsAdded = 0;
```

Comment:
Not sure if this var is actually served as a boolean?

Reply:
Is this var used in L108. Not sure why you defined it here?

Reply:
Removed the variable - it ended up being unnecessary based on changes I made later on.